codegen/src/test/resources/software/amazon/awssdk/codegen/poet/client/test-abstract-async-client-class.java

```diff
@@ -608,7 +621,8 @@ public CompletableFuture<StreamingInputOperationResponse> streamingInputOperatio
     public <ReturnT> CompletableFuture<ReturnT> streamingInputOutputOperation(
         StreamingInputOutputOperationRequest streamingInputOutputOperationRequest, AsyncRequestBody requestBody,
         AsyncResponseTransformer<StreamingInputOutputOperationResponse, ReturnT> asyncResponseTransformer) {
-        return delegate.streamingInputOutputOperation(streamingInputOutputOperationRequest, requestBody);
+        return delegate
+            .streamingInputOutputOperation(streamingInputOutputOperationRequest, requestBody, asyncResponseTransformer);
```

Comment:
Oops, looks like some of these were broken before?

Reply:
yup i noticed they were off in the diff